docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
     Use "dce [command] --help" for more information about a command.
     ```
 
-4. Type `dce init` to configure dce via an interactive prompt. This will generate a config file at ~/.dce.yaml
+1. Type `dce init` to generate a new config file at ~/.dce.yaml. Leave everything blank for now.
+
+## Deploying DCE
+
+1. [Download and install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
```

Comment:
I would tweak this a bit. The AWS CLI isn't actually a dependency of the DCE CLI, right? We're just borrowing the `~/.aws/credentials` file to load creds.

But we can also load creds from env vars. 

Note quite sure the best wording for this. The AWS docs on this are a little messy.

Checkout how Terraform talks through it: https://www.terraform.io/docs/providers/aws/index.html#shared-credentials-file (though that's a lot more verbose than we need)

Or see the AWS CLI order of precedence:
https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#config-settings-and-precedence

Note that we also have another way to load AWS creds, from the `api.token` config in dce.yml, which is populated by the `dce auth` command.

I'm thinking maybe we just need a separate section or page to cover AWS creds.

Reply:
Unfortunately we are using the aws cli as a dependency [here](https://github.com/Optum/dce-cli/blob/master/internal/util/aws.go#L116).

I think we discussed using the aws cli to manage creds in a way that was platform agnostic a while ago. I've since discovered [build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints) and believe we should use those instead.

Thanks for the helpful links. I'll go ahead and update this section.